openquake/hazardlib/gsim/zalachoris_rathje_2019.py

```diff
@@ -0,0 +1,404 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2015-2017 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module exports :class:'ZalachorisRathje2019'
+"""
+from __future__ import division
+import numpy as np
+from openquake.hazardlib.gsim.base import CoeffsTable
+from openquake.hazardlib import const
+from openquake.hazardlib.imt import PGA, PGV, SA
+from openquake.hazardlib.gsim.boore_2014 import BooreEtAl2014
+
+
+class ZalachorisRathje2019(BooreEtAl2014):
+    """
+    Implements the Induced Seismicity GMPE of Zalachoris & Rathje (2019)
+    for Texas, Oklahoma and Kansas.
+    Ground Motion Model for Small-to-Moderate Earthquakes
+    in Texas, Oklahoma, and Kansas. Earthquake Spectra.
+    Data: 4528 observations from 376 EQs
+    of Mw 3.0-5.8 in Texas, Oklahoma and Kansas,
+    distances between 4 and 500km. PGA, PGV, SA(0-10s)
+    version: Apr 28, 2020. Verified with margin 1%
+    """
+    #: The GMPE is derived from induced earthquakes
+    DEFINED_FOR_TECTONIC_REGION_TYPE = const.TRT.INDUCED
+
+    #: Supported intensity measure types are spectral acceleration,
+    #: peak ground velocity and peak ground acceleration
+    DEFINED_FOR_INTENSITY_MEASURE_TYPES = set([
+        PGA,
+        PGV,
+        SA
+    ])
+
```

Comment:
We are using the set syntax everywhere now, i.e. `DEFINED_FOR_INTENSITY_MEASURE_TYPES = {PGA. PGV, SA}`. Same for similar definitions below.